internal/application/filesystem.go

```diff
@@ -507,9 +508,16 @@ func runTranslate(appName string, artifactpath string, translatech chan string,
 		close(outch)
 		m2kqaservermetadatapath := filepath.Join(artifactpath, m2kQAServerMetadataFile)
 		os.RemoveAll(m2kqaservermetadatapath)
+		srcDirectoryPath := filepath.Join(appName, assetsDirectory, srcDirectory)
```

Comment:
Does this handle the scenario referred to by the below comment in https://github.com/konveyor/move2kube/blob/9a6767f2b283ed99404979f6ed4e2f4c52dec416/internal/transformer/templates/K8sReadme.md file? If so, we should possibly remove the warning from Move2Kube. Also, if we are going to have the zip with the src folder, we have to think through whether we need to give them the copysources script and steps in readme.

Also, possibly we should provide for a way for users to know which files have changed/added.

@HarikrishnanBalagopal Thoughts based on when you added the below comment?

{{if .AddCopySourcesWarning }} IMPORTANT!!: If you used the UI for translation then "./scripts/copysources.sh" may copy to the wrong folder. This is a limitation of the beta version. It should be fixed in future versions of move2kube-ui.
In the meantime you can either:
copy the sources to the appropriate folders inside "./containers/" manually. "./containers/" has the same folder structure as the sources folder so simply copy the appropriate source files/folders to the corresponding folders inside "./containers/".
move the sources into a directory with the same name and then try copysources with that.
Example: if sources is a folder called "foo" you might try moving it into "foo/foo/foo" and then doing "./scripts/copysources.sh path/to/foo/". This will require you to read "./scripts/copysource.sh" and have some knowledge of how "cp -r" works in order to get it right. {{ end }}

Reply:
By running `./scripts/copysources.sh path/to/src/` it copies the appropriate source files/folders to the corresponding folders inside "./containers/".